CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # avro-builder changelog
 
+## v0.16.0
+- Add support for providing an explicit `namespace` on `extend`. Default lookup 
```

Comment:
Minor, but consider splitting this as two separate bullets?

Reply:
👍